pkg/k8s/slim/k8s/apiextensions-client/clientset/versioned/clientset.go

```diff
@@ -1,5 +1,16 @@
-// SPDX-License-Identifier: Apache-2.0
 // Copyright 2017-2021 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
```

Comment:
I imagine this license header is sourced from somewhere. Could we fix the source for this generated file?